src/components/autocompleteInput.js

```diff
@@ -622,6 +623,36 @@
 
     const currentValue = getValue();
 
+    useEffect(() => {
+      let debounceCurrentValue;
+
+      if (!isListProperty) {
+        if (currentValue !== debouncedCurrentValue) {
+          debounceCurrentValue = setTimeout(() => {
+            setDebouncedCurrentValue(currentValue);
+          }, 250);
+        } else {
+          let triggerEventValue;
+
+          if (!isListProperty) {
+            triggerEventValue = debounceCurrentValue
+              ? debounceCurrentValue[valueProp.name]
+              : '';
+          } else if (isListProperty) {
```

Comment:
We can never enter this else-if statement.
If `isListProperty` is false, the conditional-check on line 629 is valid. 
And now we are checking: 
`else if(false)` which ofcourse does not work.

Reply:
So this can be removed?

Reply:
I have removed the first check since we want to use the onchange interaction with the list property and relation property